### PR TITLE
Created custom text input with ipsum generator

### PR DESCRIPTION
**Pull request by jeroenfurst**
### Motivation

I have created this custom element for the contest. It helped me understand the concept of creating custom Kentico Cloud elements and I look forward to creating more in the future.

### Checklist

- [x] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Simply follow the instructions. Optionally you can pass in the length argument to specify the number of words that it needs to generate.

### CODE REVIEW

**Comment by Simply007** [comment]
You could check https://github.com/Kentico/custom-element-samples/blob/master/CountryStateSelector/README.md for inspiration ;)

**Comment by jeroenfurst** [comment]
Done @Simply007!

File: IpsumInput/ipsum-input-component.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Input with Ipsum Generator</title>
+
+    <!-- libs -->
+    <script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
+    <script src="lorem.js"></script>
+
+    <!-- Kentico Custom element CSS-->
+    <link rel="stylesheet" type="text/css" href="https://kentico.github.io/custom-element-samples/shared/custom-element.css">
```
Comment (Simply007) [inline IpsumInput/ipsum-input-component.html]: Would it be possible to store the css locally in the package and link it as a relative resource `./custom-element.css`?

File: IpsumInput/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* In Kentico Cloud open the Content types tab
+* Create or edit a content model to which you want to add the ipsum input
+* Add the **Custom element** content element
+* Configure the content element
+* Use the following URL as Hosted code URL (HTTPS): https://kentico.github.io/custom-element-samples/IpsumInput/ipsum-input-component.html
+* Optional: Provide the following JSON parameters for the custom element to configure the number of words to be generated
+
+```json
+{
+    "Length": "<NUMBER OF WORDS>"
```
Comment (Simply007) [inline IpsumInput/README.md]: I would show that the type in json is a number.

```suggestion
    "Length": 5 // NUMBER OF WORDS
```

**Comment by Simply007** [comment]
> Nice suggestions @Simply007! I was thinking the same, nice to see you already implemented them 👍

Actually, I have just suggested the HTML.
If you are happy with the suggestion I would be glad to approve this pull request after it is implemented.

**Comment by jeroenfurst** [comment]
Done @Simply007!

File: IpsumInput/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* In Kentico Cloud open the Content types tab
+* Create or edit a content model to which you want to add the ipsum input
+* Add the **Custom element** content element
+* Configure the content element
+* Use the following URL as Hosted code URL (HTTPS): https://kentico.github.io/custom-element-samples/IpsumInput/ipsum-input-component.html
+* Optional: Provide the following JSON parameters for the custom element to configure the number of words to be generated
+
+```json
+{
+    "Words": 5 // NUMBER OF WORDS
```
Comment (Simply007) [inline IpsumInput/README.md]: Change case of the sample json
```suggestion
    "words": 5 // NUMBER OF WORDS
```

**Comment by jeroenfurst** [comment]
I closed the pull request. Do you still need to merge it?

Met vriendelijke groet,

Jeroen Fürst
Kentico MVP & Solutions Architect

Aanwezig op: maandag, dinsdag, donderdag en vrijdag


jeroen.furst@truelime.nl<mailto:jeroen.furst@truelime.nl>
T:  +31 (0)76 820 0920
Charles Petitweg 35 B13
4827 HJ Breda

[http://www.truelime.nl/TrueLime/media/TrueLime/handtekening_outlook/truelime.png]<https://www.truelime.nl>

[http://www.truelime.nl/getmedia/c21b677d-a539-41d4-a69f-ddd02ac96c9b/emailtemplate_linkedin_tlgrijs.aspx]<https://www.linkedin.com/company/255205>  •   [http://www.truelime.nl/getmedia/e3a7d014-b393-403f-a62f-31e7237fdc5d/emailtemplate_fb_tlgrijs.aspx] <http://www.facebook.com/truelimeNL>   •   [http://www.truelime.nl/getmedia/ae607b70-363a-45d1-bb1e-419ddc27d43c/emailtemplate_twitter_tlgrijs.aspx] <http://www.twitter.com/truelimeNL>
Van: Ondřej Chrastina <notifications@github.com>
Verzonden: dinsdag 2 april 2019 11:55
Aan: Kentico/custom-element-samples <custom-element-samples@noreply.github.com>
CC: Jeroen Fürst <jeroen.furst@truelime.nl>; Mention <mention@noreply.github.com>
Onderwerp: Re: [Kentico/custom-element-samples] Created custom text input with ipsum generator (#26)


@Simply007 approved this pull request.

**Comment by Simply007** [comment]
Yes, I wanted to do it after final check, then I just needed to go... you know something urgent came up :-)

**Pull request merged**